tc/sgx/trusted_worker_manager/enclave_wrapper/avalon_enclave_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,145 @@
+# limitations under the License.
+
+CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
+
+PROJECT(avalon-enclave-bridge C CXX)
+
+LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+SET (EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
+SET (LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/lib)
+
+SET(DEPS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/deps")
+SET(TCF_TOP_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../../../")
+SET(TCF_CORE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../")
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# Configure Environment
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+SET(SGX_MODE "$ENV{SGX_MODE}")
+if("${SGX_MODE} " STREQUAL " ")
+    SET(SGX_MODE "SIM")
+    message(STATUS "Setting default SGX_MODE=${SGX_MODE}")
+endif()
+IF("${SGX_MODE}" STREQUAL "SIM")
+    OPTION(SGX_USE_SIMULATOR "Use the sgx simulator" TRUE)
+ELSE()
+    OPTION(SGX_USE_SIMULATOR "Use the sgx simulator" FALSE)
+ENDIF()
+
+SET(SGX_SDK "$ENV{SGX_SDK}")
+if("${SGX_SDK} " STREQUAL " ")
+    message(FATAL_ERROR "SGX_SDK environment variable not defined!")
+endif()
+SET(SGX_SSL "$ENV{SGX_SSL}")
+if("${SGX_SSL} " STREQUAL " ")
+    SET(SGX_SSL "/opt/intel/sgxssl")
+    message(STATUS "Setting default SGX_SSL=${SGX_SSL}")
+endif()
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# ADDITIONAL LIBRARIES
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+# sgx edger is a utility that generates edl artifacts
+SET(SGX_EDGER "${SGX_SDK}/bin/x64/sgx_edger8r")
+
+SET(SGX_INCLUDE "${SGX_SDK}/include")
+SET(SGX_SSL_INCLUDE "${SGX_SSL}/include")
+
+SET(SGX_LIBS_UNTRUSTED_NAMES sgx_ukey_exchange)
+
+SET(SGX_LIBS_UNTRUSTED_NAMES ${SGX_LIBS_UNTRUSTED_NAMES} pthread)
+
+IF(${SGX_USE_SIMULATOR})
+    ADD_DEFINITIONS(-D SGX_SIMULATOR)
+    SET(SGX_LIBS_UNTRUSTED_NAMES sgx_urts_sim sgx_uae_service_sim ${SGX_LIBS_UNTRUSTED_NAMES})
+ELSE()
+    SET(SGX_LIBS_UNTRUSTED_NAMES sgx_urts sgx_uae_service ${SGX_LIBS_UNTRUSTED_NAMES})
+ENDIF()
+
+FOREACH(lib ${SGX_LIBS_UNTRUSTED_NAMES})
+    SET(${lib} ${lib})
+ENDFOREACH(lib)
+
+FOREACH(lib ${SGX_LIBS_UNTRUSTED_NAMES})
+    SET(SGX_LIBS_UNTRUSTED ${SGX_LIBS_UNTRUSTED} ${${lib}})
+ENDFOREACH(lib)
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+FUNCTION(SGX_EDGE_UNTRUSTED EDL EDGE_FILES)
+    GET_FILENAME_COMPONENT(EDL_BASE_NAME ${EDL} NAME_WE)
+    GET_FILENAME_COMPONENT(EDL_DIR_NAME ${EDL} DIRECTORY)
+    INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+
+    SET (EDGE_FILES_LIST "${CMAKE_CURRENT_BINARY_DIR}/${EDL_BASE_NAME}_u.h" "${CMAKE_CURRENT_BINARY_DIR}/${EDL_BASE_NAME}_u.c")
+    SET (${EDGE_FILES} ${EDGE_FILES_LIST} PARENT_SCOPE)
+    ADD_CUSTOM_COMMAND( OUTPUT ${EDGE_FILES_LIST}
+        COMMAND "${SGX_EDGER}" --untrusted ${EDL} --search-path ${SGX_INCLUDE} --search-path ${SGX_SSL_INCLUDE} --search-path ${EDL_DIR_NAME}
```

Comment:
These long lines at line 91 and 88 are hard to read.
You can break them up by with a backslash (`\`).  For example:

`COMMAND "${SGX_EDGER}" --untrusted ${EDL} --search-path ${SGX_INCLUDE}\`
`     --search-path ${SGX_SSL_INCLUDE} --search-path ${EDL_DIR_NAME}`

Reply:
Done